main.go

```diff
@@ -1,6 +1,13 @@
 package main
 
+import "flag"
+
+var cutefood = true
+
 func main() {
+	flag.BoolVar(&cutefood, "cutefood", true, "display a cute character as food")
```

Comment:
I think we can use a more generic flag here, this does not actually say much. We should use something like `allow-unicode`, that will give us more flexibility to use unicode for other things that are not the `food`